examples/hosts/app-integration/external-data/tests/externalDataService.test.ts

```diff
@@ -37,11 +46,23 @@ describe("mock-external-data-service", () => {
 	 */
 	let externalDataService: Server | undefined;
 
+	/**
+	 * Datastore mapping of external resource id to its subscribers.
+	 *
+	 * @defaultValue A new new map will be initialized.
+	 */
+	let webhookCollection: Map<string, MockWebhook<ITaskData>>;
+
 	beforeEach(async () => {
 		externalDataSource = new ExternalDataSource();
+		webhookCollection = new Map<string, MockWebhook<ITaskData>>();
+		const webhook = new MockWebhook();
+		webhook.registerSubscriber("https://www.fluidframework.com");
+		webhookCollection.set("task-list-1", webhook);
```

Comment:
Same as in the other comment, do we need this subscription for all tests?

Reply:
We need it for the set-task tests and the problem is there is no way to access the webhook subscription outside of this entry point.